server/s3_object_metadata.cc

```diff
@@ -819,6 +827,10 @@ std::string S3ObjectMetadata::to_json() {
     root["System-Defined"]["x-amz-delete-marker"] = "true";
   }
 
+  if (_null) {
+    root["is-null"] = "true";
```

Comment:
Sorry, I should have commented earlier. This new field is not there in the HLD, the HLD update will be needed.

Reply:
I don't have the write access but I have added a [comment](https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/562593803/Versioning?focusedCommentId=847676084#comment-847676084) on the HLD document for the same.